argusclient/client.py

```diff
@@ -194,6 +196,12 @@ def values(self):
         self._init_all()
         return self._coll.values()
 
+    def set_get_all_path(self, path):
+        self.get_all_path = path
+
+    def set_get_all_params(self, params):
+        self.get_all_params = params
```

Comment:
Are these setters meant only for the testing? If so, shouldn't you just mock them?

Reply:
No, these are used to set the parameters of the endpoint call. Seemed like the best way was with a setter. Also since theres a setter for get_all_path

Reply:
OK, I started another thread, so resolving this.